tests/unit-tests/processors/models/processorsManager-test.js

```diff
@@ -177,5 +214,37 @@ describe('Processor manager tests', function () {
             should(exception.statusCode).eql(422);
             should(exception.message).eql('javascript syntax validation failed with error: Unexpected identifier');
         });
+
```

Comment:
duplicate tests ? 
should fail - updating a processor name to another existing processor name

should throw an error updating processor to name already exists

Reply:
done